Assets/Plugins/Android/AndroidManifest.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.shopify.unity.buy">
+    <application android:label="@string/app_name" android:supportsRtl="true">
+        <meta-data
+            android:name="com.google.android.gms.wallet.api.enabled"
+            android:value="true" />
+
+        <activity android:name="com.shopify.unity.buy.ShopifyUnityPlayerActivity"
```

Comment:
The custom launch activity requires you to add it to the AndroidManifest.xml for the plugin so I also moved the wallet API requirement to here as well.